<ui version="4.0" >
 <class>BookmarkManage</class>
 <widget class="QDialog" name="BookmarkManage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manage Bookmarks</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QListView" name="listView" >
       <property name="dragDropMode" >
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bookmarkButtonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::NoButton</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Host:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="host" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Room:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="room" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Nickname:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="nickname" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Password:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="password" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QCheckBox" name="autoJoin" >
       <property name="text" >
        <string>Auto-join</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>66</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookmarkManage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>475</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel" >
     <x>454</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookmarkManage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>384</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel" >
     <x>352</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
